drivers/media/i2c/hm2172.c

```diff
@@ -910,10 +921,20 @@ struct hm2172 {
 	struct v4l2_ctrl *hblank;
 	struct v4l2_ctrl *exposure;
 
-	struct clk *img_clk;
-	struct regulator *avdd;
+	/* GPIO for reset */
 	struct gpio_desc *reset;
+	/* GPIO for Lattice handshake */
 	struct gpio_desc *handshake;
+	/* regulator */
```

Comment:
The comment does not add anything useful.

Reply:
Removed